### ISSUE TITLE
Card Viewing

### ISSUE THREAD

**Issue by kernelkaribou** [BaldissaraMatheus/Tasks.md#30]
### Suggestion

Currently cards when clicked on will automatically open to editor mode. This requires formatting like links and other markdown styles generally useless unless you select the preview icon.

One idea would be to cards will automatically open in "preview"/read only mode with a simple icon to change to edit mode. That way any additional views will be reader formatting and remove  the various shortcuts for a clean look (similar to writing this right now in Github)

On Click example:

![image](https://user-images.githubusercontent.com/25406580/226719648-8b197f1c-6f2b-4c40-90fa-1373f29586d3.png)

After entering edit
![image](https://user-images.githubusercontent.com/25406580/226719296-e6452a5a-3999-4805-afd8-4d55debe5d0b.png)

If there is no content it could be smart enough to enter edit mode automatically.

**Comment by BaldissaraMatheus** [comment]
I could do that but I think it would become annoying having to click to enable editor mode to be able to change the text, and I think opening the card in a different mode when the content has changed could feel a bit inconsistent. I really wanted the links to open while in editor mode, but the current editor (SimpleMDE) is pretty limiting.

I'm thinking about replacing the SimpleMDE with a WYSIWYG editor, so it's always legible while fully interactable. I've researched a bit for options and the best ones I found so far are [slate](https://github.com/ianstormtaylor/slate) and [tiptap](https://github.com/ueberdosis/tiptap), but I still plan to search for more options and experiment with them
